x/smart-account/ante/ante_test.go

```diff
@@ -187,13 +188,16 @@ func (s *AuthenticatorAnteSuite) TestSignatureVerificationWithAuthenticatorInSto
 }
 
 // TestSignatureVerificationOutOfGas tests that the ante handler exits early by running out of gas if the
-// fee payer has not been authenticated before consuming the 20_000 low gas limit (even if the specified limit is 300k)
+// fee payer has not been authenticated before consuming the parametrized max unauthenticated gas limit (even if the specified limit is 300k)
```

Comment:
Note: took me a while to process this test by referring to the hardcoded values (also I think 20k was wrong - it is 120k). Instead, changed to using the parameter value directly for clarity

Reply:
Yeah that's perfect thank you!